smartconfig/entry.py

```diff
@@ -97,7 +97,7 @@ def __init__(cls, name: str, bases: Tuple[type, ...], dict_: Dict[str, Any], pat
         Initialize the new entry.
 
         Raises:
-            DuplicateConfiguration: An entry is already registered for this path, use the `path` metaclass argument.
+            PathConflict: An entry is already registered for this path, use the `path` metaclass argument.
```

Comment:
This is a comma splice. Use a semicolon to separate independent clauses or use a full stop to make them separate sentences.